src/content/docs/logs/ui-data/lookup-tables-ui.mdx

```diff
@@ -52,6 +52,7 @@ Some more details about lookup tables:
 * Tables can only be queried from the [account](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#organization-accounts) in which the table was uploaded. 
 * Each [account](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#organization-accounts) has a limit of 20 lookup tables.
 * Lookup table data can't be used in NRQL alert conditions. 
+* Lookup tables have indefinite data retention by default.
```

Comment:
what does 'indefinite' mean? this struck me as a bit unclear. if it exists forever, could we say that?

Reply:
Indefinite meaning we retain the data in lookup tables forever. We could absolutely rephrase it to say we retain lookup tables forever if that's clearer @zuluecho9.

Reply:
Hey @zuluecho9 is there anything I should change with the wording above? Should we not use indefinite to describe the data retention?